src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -33,10 +33,7 @@
 
 namespace Microsoft.ML.Transforms.Conversions
 {
-    /// <summary>
-    /// The ValueMappingEstimator is a 1-1 mapping from a key to value. This particular class load the mappings from an <see cref="IDataView"/>.
-    /// This gives user the flexibility to load the mapping from file instead of using IEnumerable in <see cref="ValueMappingEstimator{TKey, TValue}"/>
-    /// </summary>
+    /// <include file='doc.xml' path='doc/members/member[@name="ValueMappingEstimator"]/*' />
     public class ValueMappingEstimator : TrivialEstimator<ValueMappingTransformer>
```

Comment:
>ValueMappingEstimator [](start = 17, length = 21)

i'd put it in the transformer too.

Reply:
put the comments for the estimator on the transformer? Or create a new entry for the transformer?

---
In reply to: [252889880](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252889880) [](ancestors = 252889880)

Reply:
Added to transformer.

---
In reply to: [252937283](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252937283) [](ancestors = 252937283,252889880)